src/components/supervisor.tsx

```diff
@@ -52,8 +50,8 @@ export function Supervisor({
             </div>
           </AccordionTrigger>
           <AccordionContent className="space-y-2 p-0">
```

Comment:
**suggestion:** I think the gap between accordion's content is bigger in our design. Increasing the spacing value makes the component more readable
```suggestion
          <AccordionContent className="space-y-6 p-0">
```

*i don't know how i missed all this in supervisor component pr itself 😭*